akanda/nova/neutronv2/api.py

```diff
@@ -61,55 +64,72 @@ def allocate_for_instance(self, context, instance, **kwargs):
             # pre-allocated port we also remove it from this set.
             available_macs = set(hypervisor_macs)
         neutron = neutronv2.get_client(context)
-        LOG.debug(_('allocate_for_instance() for %s'),
-                  instance['display_name'])
-        if not instance['project_id']:
+        LOG.debug('allocate_for_instance()', instance=instance)
```

Comment:
The remaining translation wrappers are used for LOG.info and LOG.warning which should be still in use in Openstack I believe.